### ISSUE TITLE
Bug: Error messages from TTS engine

### ISSUE THREAD

**Issue by jofemodo** [zynthian/zynthian-issue-tracking#1699]
**Describe the bug**
The UI log shows these messages:

>TTS error: expected str, bytes or os.PathLike object, not NoneType

**To Reproduce**
1. Enable TTS without a dedicated soundcard
2. Do some UI interaction
3. See log messages

**Expected behavior**
TTS remains disabled. No TTS erros are shown.

**Setup:**
 - V5.1
 - Vangelis branch

**Comment by riban-bw** [comment]
There is an error when no sound card exists described by @ch000:
```
Jun 27 19:36:39 zynthian startx[882]: ALSA lib confmisc.c:165:(snd_config_get_card) Cannot get card index for None
Jun 27 19:36:39 zynthian startx[882]: ERROR:zynthian_tts._do_beep: TTS failed to send tone to soundcard - No such device [hw:None]
```

We need to avoid access to non-existing soundcards whilst supporting their temporary absence, i.e. we should not disable TTS or remove the soundcard config just because the USB soundcard is not connected or switched on.

**Comment by riban-bw** [comment]
I manged to reproduce this by:

- Connect a USB soundcard and assign as ZynVoice TTS device.
- Remove the USB soundcard.
- Reboot.

Reasserting the soundcard selection in Admin menu, ZynVoice options fixes the issue. Sometimes the soundcard is shown selected when it is not. In effect, zynvoice thinks it has no soundcard, even when it does. The background checks for soundcard may need to be improved.
